aten/src/ATen/native/native_functions.yaml

```diff
@@ -982,6 +982,7 @@
   dispatch:
     CPU: resize_cpu_
     CUDA: resize_cuda_
+    QuantizedCPU: resize_cpu_
```

Comment:
hm, can you check that it throws if the input quantizer is per-channel? It's not safe to invoke it in this case

Reply:
I think I can but it probably involves making a specific quantized resize function rather than reusing this one